### DISCUSSION TITLE
Centering equations inside enum

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3074] [Q&A]
When using:

```
#enum(numbering: "a)",
  enum.item([
  Tem-se:
  $ integral x d x &=x^2 \
  &= x^2 $]))
```

The equation does not get centered:

![image](https://github.com/typst/typst/assets/75945439/ecad6b41-1c64-49f2-a7e1-a648d351781e)

How can we center it?

**Comment by Doublonmousse** [comment]
See here to force equations to be centered 
https://github.com/typst/typst/issues/529#issuecomment-1703651557

**Reply by psads-git** [reply]
Thanks, @Doublonmousse!

**Reply by Doublonmousse** [reply]
Maybe add `breakable: true` as well, just to be safe. (If this is set to false, you may end up with blank space to force the list to stay in one single block on the next page rather than be broken and continue on the next page)

**Comment by emilyyyylime** [comment]
Another fix I use is just `#h(1fr)` to force the enum to 'expand'
